### DISCUSSION TITLE
How to add a hash value to a ConfigMap suffix in Tanka

### DISCUSSION THREAD

**Discussion by khirotaka** [grafana/tanka#1163] [Q&A]
Hello!

I am currently testing whether I can migrate an existing project that uses Kustomize to Tanka.

When I generate a ConfigMap using Kustomize's `configMapGenerator`, a hash value is added to the suffix. Is it possible to achieve the same thing in Tanka?

I'm aware that there's a way to use `configMapVolumeMount()`, but I would like to know if there are any other methods available.

Thanks

**Comment by zerok** [comment]
Hi 🙂  Haven't seen anything else yet. Personally, I tend to look at things like https://github.com/stakater/reloader to handle change detection on secrets and configmaps.
